app/main.py

```diff
@@ -1,3 +1,7 @@
+from collections import Counter
+from subprocess import check_output
+
+
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    return Counter(phrase.lower())[letter.lower()]
+
```

Comment:
Please fix the problem with the tests + the solution with Counter() is a bit overkill, you can limit yourself to the option with .count().